portfolio/src/main/webapp/splashPage.js

```diff
@@ -12,41 +12,77 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const TILES = {"hiking":8, "me":9, "projects":14, "travel":8};
+const TILES = {"travel": 8, "projects": 14, "me": 9, "hiking": 8};
 
-
-/** 
- * Load a random pic for a specific splash page icon
+/**
+ * return a random img with index i from the proper subdirectory
  */
-function randomPic(filePrefix, numFiles, elId) {
-  var picOptions = makeFileArray(filePrefix, numFiles); //create arr of options
-  randInd = Math.floor((Math.random() * picOptions.length));
+function randomImg(tileName) {
+  var randInd = Math.floor((Math.random() * TILES[tileName]));
+  var path = `images/${tileName}_images/${tileName}_${randInd+1}.jpeg`;
```

Comment:
this feels like confusing things.  Choose 0-index or 1-index and make it consistent everywhere.